src/main/kotlin/me/nobaboy/nobaaddons/config/categories/InventoryCategory.kt

```diff
@@ -129,19 +129,19 @@ object InventoryCategory {
 			)
 			tickBox(
 				tr("nobaaddons.config.inventory.slotInfo.petLevel", "Pet Level"),
-				tr("nobaaddons.config.inventory.slotInfo.petLevel.tooltip", "Shows the level of a non-maxed pet"),
+				tr("nobaaddons.config.inventory.slotInfo.petLevel.tooltip", "Displays the level of a non-maxed level pet"),
 				default = defaults.inventory.slotInfo.petLevel,
 				property = config.inventory.slotInfo::petLevel
 			)
 			tickBox(
 				tr("nobaaddons.config.inventory.slotInfo.petItem", "Pet Items"),
-				tr("nobaaddons.config.inventory.slotInfo.petItem.tooltip", "Shows an icon for certain pet items, like EXP Share or Lucky Clover"),
+				tr("nobaaddons.config.inventory.slotInfo.petItem.tooltip", "Displays an icon for specific pet items, such as EXP Share, Lucky Clover Core or Tier Boost"),
```

Comment:
```suggestion
				tr("nobaaddons.config.inventory.slotInfo.petItem.tooltip", "Displays an icon for specific pet items, such as EXP Share, Lucky Clover, or Tier Boost"),
```

lucky clover core isn't a pet item, and use oxford comma for clearer separation